lib/rinha/transactions/transaction.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Rinha.Transactions.Transaction do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias Rinha.Customers.Customer
+
+  @primary_key false
+  embedded_schema do
+    field(:id, :integer)
+    field(:amount, :integer)
+    field(:customer_id, :integer)
+    field(:type, :string)
```

Comment:
dá pra definir como um enum aqui, acho que ajuda a ficar explícito o que esse campo aceita e aí dá pra remover o validate_inclusion ali embaixo, o que acha?

```suggestion
    field(:type, Ecto.Enum, values: [:c, :d])
```

Reply:
@norbajunior melhor mesmo, fica mais explícito

Reply:
feito